server/backend/tests/conftest.py

```
@@ -39,131 +59,70 @@ def _build_settings(db_path: Path) -> Settings:
-    async def create_api_key(self, *args, **kwargs):
-        return await self.api_keys.create(*args, **kwargs)
 
-    async def get_active_api_key_by_id(self, key_id: str):
-        return await self.api_keys.get_active_by_id(key_id)
+@pytest_asyncio.fixture
+async def users_repo(repos: _RepoBundle) -> UserRepository:
+    """
+    Provide the UserRepository instance from the shared repository bundle.
+    
+    Parameters:
+    	repos (_RepoBundle): Shared bundle containing repository instances for tests.
```

Ruff (0.15.15)

[error] Indentation contains mixed spaces and tabs

(E101)